libs/formAnswers.js

```diff
@@ -103,24 +103,31 @@ function getMultipleFieldAnswers(field, previousAnswers) {
   return answers;
 }
 
-function getInitialValue(field, user, previousAnswers) {
+function getInitialValue(field, users, previousAnswers) {
   let initialValue;
 
   field.loadPrevious.forEach(matchString => {
+    let user;
     const strArray = matchString.split('.');
-    if (strArray[0] === 'user') {
-      initialValue = getUserInfo(user, strArray.slice(1)) || initialValue;
-      if (initialValue) {
-        return formatAnswer(field.id, field.tags, initialValue);
-      }
+    switch (strArray[0]) {
+      case 'user':
+        user = users.find(element => element.role === 'applicant');
+        initialValue = getUserInfo(user, strArray.slice(1)) || initialValue;
+        break;
+      case 'coApplicant':
+        user = users.find(element => element.role === 'coapplicant');
+        initialValue = getUserInfo(user, strArray.slice(1)) || initialValue;
+        break;
+      default:
+        initialValue = getCaseAnswer(previousAnswers, matchString) || initialValue;
+        break;
```

Comment:
This section is a bit repetitive, as each switch `case` only really checks for a different role. Could it be rewritten to reduce code duplication? That would be useful for when we add future cases as well.

Suggestion (untested code):

```javascript
    const strArray = matchString.split('.');

    const userRoleMap = {
      user: 'applicant',
      coApplicant: 'coapplicant',
    };

    const role = userRoleMap[strArray[0]];

    if (role) {
      const user = users.find(element => element.role === role);
      initialValue = getUserInfo(user, strArray.slice(1)) || initialValue;
    } else {
      initialValue = getCaseAnswer(previousAnswers, matchString) || initialValue;
    }
```